cmd/project/samples.go

```diff
@@ -85,3 +105,39 @@ func runSamplesCommand(clients *shared.ClientFactory, cmd *cobra.Command, args [
 	// Execute the `create` command with the set flag
 	return createCmd.ExecuteContext(ctx)
 }
+
+// listSampleSelection outputs available samples matching a language flag filter
+func listSampleSelection(ctx context.Context, clients *shared.ClientFactory, sampleRepos []create.GithubRepo) error {
+	filteredRepos := filterRepos(sampleRepos, samplesLanguageFlag)
+	sortedRepos := sortRepos(filteredRepos)
+	message := ""
+	if samplesLanguageFlag != "" {
+		message = fmt.Sprintf("Listing %d \"%s\" project samples", len(sortedRepos), samplesLanguageFlag)
+	} else {
+		message = fmt.Sprintf("Listing %d project samples", len(sortedRepos))
+	}
+	clients.IO.PrintInfo(ctx, false, "\n%s", style.Sectionf(style.TextSection{
+		Emoji: "house_buildings",
+		Text:  "Samples",
+		Secondary: []string{
+			message,
+		},
+	}))
+	emojis := []string{
+		"microscope",
+		"test_tube",
+		"petri_dish",
+		"dna",
+	}
```

Comment:
thought: While the emojis are a nice touch, I found them a little confusing. I spent some time trying to understand the meaning behind the emoji. I think it would be better to not include the emojis.

Reply:
Thanks for taking the time to look back into this! Since the `samples` command has low usage, I think we can start by including the lengthy descriptions and revisit it later.